docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: Guide on working with Etsy in Gatsby
```

Comment:
```suggestion
Title: Working with Etsy in Gatsby
```
The title should be able to focus on what technologies are in use without worrying about what kind of document it is.